Marksman/Refs.fs

```diff
@@ -316,6 +334,7 @@ module Dest =
                 | Dest.LinkDef _ -> [ srcDoc ]
                 | Dest.Heading _
                 | Dest.Doc _ -> Folder.docs folder |> List.ofSeq
+                | Dest.Tag _ -> [ srcDoc ]
```

Comment:
Similar to above, this is the list of docs where you should search for references. So, you could do something like this:
```
| Dest.Doc _
| Dest.Tag _ -> Folder.docs folder |> List.ofSeq
```

Because you want to search for references in the whole folder.